README.md

```diff
@@ -14,47 +23,127 @@ And then execute:
 
     $ bundle
 
-Or install it yourself as:
+Or execute the following line to install the gem without bundler.
 
     $ gem install ricohapi-mstorage
 
 ## Usage
 
 You can find [a working rails sample app using Ricoh Media Storage API](https://github.com/ricohapi/media-storage-sample-app).
 
-### Obtaining Access Token for Media Storage API
+## SDK API
 
-To obtain access tokens, you can use [ricohapi-auth gem](https://github.com/ricohapi/auth-rb).
+### Obtain an access token
+
+```ruby
+client = RicohAPI::Auth::Client.new(
+  '<your-client-id>',
+  '<your-client-secret>'
+)
```

Comment:
I think the following code better `lient = RicohAPI::Auth::Client.new '<your-client-id>', '<your-client-secret>'` than above code.